src/blockchain/consensus.md

```diff
@@ -0,0 +1,36 @@
+# Consensus
+
+Each time you send a transaction to Iroha, it gets put into a queue. When
+it's time to produce a new block, the queue is emptied, and the consensus
+process begins. This process is equal parts common sense and black
+magic[^1].
+
+The mundane aspect is that a special set of peers needs to take the
+transaction queue and reproduce the same world state. If the world state
+cannot be reproduced for some reason or another, none of the transactions
+get committed to a block.
+
+The consensus starts over from scratch by choosing a different special set
+of peers. This is where the black magic comes in. There is a number of
+things that are fine-tuned: the number of peers in the voting process, the
+way in which subsequent voting peers are chosen, and the way in which the
+peers communicate that consensus has failed. Because this changes the view
+of the world, the process is called a _view change_. The exact reason for
+why the view was changed is encoded in the _view change proof_, but
+decoding that information is an advanced topic that we won't cover here.
+
+The reasoning behind this algorithm is simple: if someone had some evil
+peers and connected them to the existing network, if they tried to fake
```

Comment:
it's another simplification, jfyi, in a permissioned blockchain 'someone' cannot attach new peers, those are approved via processing a legitimate transaction including an instruction to add the peer